CodeEditModules/Modules/WorkspaceClient/src/UI/FileIcon.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  FileIcon.swift
+//  
+//
+//  Created by Nanashi Li on 2022/05/06.
+//
+
+import SwiftUI
+
+// swiftlint:disable function_body_length
+// swiftlint:disable cyclomatic_complexity
+public struct FileIcon {
+
+    // swiftlint:disable identifier_name
+    public enum FileType: String {
+        case json = "json"
```

Comment:
if the `rawValue` is equal to the actual case name we can omit the `rawValue` itself